cooperator/tests/cooperator_test_mixin.py

```diff
@@ -133,7 +98,7 @@ def create_payment_account_move(self, invoice, date, amount=None):
                         0,
                         {
                             "name": invoice.payment_reference,
-                            "account_id": self.company.property_cooperator_account.id,
+                            "account_id": self.env.company.property_cooperator_account.id,
```

Comment:
Explicit is better than implicit. Let's define a `company` attribute in the set-up method that is equal to `self.env.company` so that you can use `self.company` here.

Reply:
since we’re also testing the multi-company behavior here (also possibly when calling test methods), do we really want to reference one particular company from the class? i removed the reference to it because i thought that it wasn’t useful. i’ve re-added it, we’ll see with later cases how it will turn out.